### DISCUSSION TITLE
Seq default deploy on Kubernetes MIME-type errors

### DISCUSSION THREAD

**Discussion by vkx86** [datalust/seq-tickets#1292] [Help] [answered]
Hi,
I'm using Seq locally on development machine.
Installed Seq on Kubernetes cluster using official instructions:
https://docs.datalust.co/docs/using-helm
Asked cluster admin to open public access and got MIME-type check errors in browser when accessing Seq instance.
Any clue what's wrong?

![seq-k8s-deploy-error](https://user-images.githubusercontent.com/4396252/128897860-c790cf24-22d9-401f-bac7-53bba1fe8e43.jpg)

**Comment by nblumhardt** [comment, marked as answer]
Hi! Any chance you're running an old K8s or nginx ingress version? https://github.com/kubernetes/ingress-nginx/issues/5265 seems to suggest the problem is somewhere on the ingress side. Let me know if this helps.

**Reply by vkx86** [reply]
Hi,
Your suggestion was spot on, thanks!
